CONTRIBUTING.md

```diff
@@ -1,119 +1,87 @@
+4. When you are done development work, don't forget to bring everything down gracefully by executing the following command from the project root directory:
+`docker-compose down`
+
+### Option 2: Manually setting up local environment
+
+#### Prerequisites
+
+- Postgres (9.6 or higher): https://wiki.postgresql.org/wiki/Detailed_installation_guides
+- Ruby (using rbenv or rvm install version 2.6.0): https://www.ruby-lang.org/en/documentation/installation/
+- Rails: https://guides.rubyonrails.org/getting_started.html
+- Redis:
+  - Linux: https://redis.io/topics/quickstart
+  - Windows: https://redislabs.com/ebook/appendix-a/a-3-installing-on-windows/a-3-2-installing-redis-on-window/
+  - Mac: Using homebrew:
 ```
+  brew update
+  brew install redis
 
-#### Setting up local environment manually
+  # If you want it to autorun for you at startup:
+  brew services start redis
 
-##### Prerequisites
+  # if you want to run it only when working:
+  redis-server /usr/local/etc/redis.conf
 
-* PostgreSQL (9.2+) is installed and running on your local machine
+  # test if redis is running, the following command should
+  # return 'PONG':
+  redis-cli ping
+```
 
-##### Method 1: Automatic
+#### Database Setup Method 1: Automatic
 
 Run `rails db:setup`
 
-##### Method 2: Manual
+#### Database Setup Method 2: Manual
+
+##### First Time Setup
+If you are setting up your database for the first time:
 
 * Create a postgres users
   It's recommended though optional that you use a distinct user for the harvey-api database
   * Example `createuser harvey-api_development -P`
   The `-P` flag will prompt you to create a password for the new user
 * Create the database (with the owner of the database set to user just created in the last step)
   * Example `createdb -O harvey-api_development harvey-api_development`
+* Run rails structure load to create the schema: `rails db:structure:load`, otherwise you can migrate to get any new updates: `rails db:migrate`
 
-##### Ensuring you have the latest schema
-
-If you are setting up your database for the first time, run rails structure load to create the schema: `rails db:structure:load`, otherwise you can migrate to get any new updates: `rails db:migrate`
+#### Importing data
 
-##### Importing data
+##### Shelters and Needs from Production API
 
-###### Shelters and Needs from Production API
+**DO NOT RUN THIS JOB IN PRODUCTION.** Since this job pulls data from the production API, running it in production can only be counter-productive, and would likely be destructive. The `ActiveJob`s and associated Rake task `rails api:import`, which imports data for shelters and needs from the production API into the application database, is intended for use in development and test environments only.
```

Comment:
Is it possible to clarify this sentence. It is not clear what does running in production means vs. importing that is okay to use in development. I suggest also clarifying which commands are okay to use.

Reply:
I'm a bit too familiar with the technicals to see what can be done to improve further.  Can you elaborate on what is unclear about the sentence?

As far as command usage, there are no other commands in this context.